lib/FusionInventory/Agent/Task/Inventory/Virtualization/Xen.pm

```diff
@@ -8,7 +8,7 @@ use FusionInventory::Agent::Tools;
 our $runMeIfTheseChecksFailed = ["FusionInventory::Agent::Task::Inventory::Virtualization::Libvirt"];
 
 sub isEnabled {
-    return canRun('xm');
+    return canRunOK('xm') || canRunOK('xl');
 }
```

Comment:
Ok so (sorry I need understand :p) why not only test the version of the tool (like https://github.com/fusioninventory/fusioninventory-agent/blob/master/lib/FusionInventory/Agent/Task/Inventory/Virtualization/VirtualBox.pm#L16-L19)